ci_framework/roles/reproducer/templates/play.yml.j2

```diff
@@ -1,11 +1,90 @@
 ---
+{% if use_content_provider is defined and use_content_provider | bool -%}
```

Comment:
can't we get away with 

```
{% if use_content_provider | default(false) | bool %}
```

Reply:
maybe - though I tend to dislike using |default() in these kind of cases

Reply:
(updated locally, will push once I address the other points)